shop/rest/renderers.py

```diff
@@ -19,7 +19,7 @@ def render(self, data, accepted_media_type=None, renderer_context=None):
         view = renderer_context['view']
         request = renderer_context['request']
         response = renderer_context['response']
-        template_context = self.get_template_context(data, renderer_context)
+        template_context = self.get_template_context(dict(data), renderer_context)
```

Comment:
``self.get_template_context`` conditionally returns a reference on data, hence we would modify ``data`` by modifying the ``template_context``, unless we create a copy of data.